tests/lib/prepare-restore.sh

```diff
@@ -67,6 +67,10 @@ create_test_user(){
     fi
     unset owner
 
+    # Add a new line first to prevent an error which happens when
+    # the file has not new line, and we see this:
+    # syntax error, unexpected WORD, expecting END or ':' or '\n'
+    echo >> /etc/sudoers
     echo 'test ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
Nitpick, you coud do:
```
echo -e "test ALL=(ALL) NOPASSWD:ALL\n\n" >> /etc/sudoers
```

Reply:
shellcheck from edge will complain about "echo -e" because it depends on the exact shell being used what parameters it supports. Better printf

Reply:
Let's follow up with printf after this lands.